doc/start/documenting-ceph.rst

```diff
@@ -60,32 +60,33 @@ instead of compiling program source. This sequence (the sequence of building the
 Get the Source
 --------------
 
-Ceph documentation lives in the Ceph repository right alongside the Ceph source
-code under the ``ceph/doc`` directory. For details on github and Ceph,
-see :ref:`Get Involved`.
+The source of the Ceph documentation is a colletion of ReStructured Text files
+that are in the Ceph repository in the ``ceph/doc`` directory.  For details
+on github and Ceph, see :ref:`Get Involved`.
 
 The most common way to make contributions is to use the `Fork and Pull`_
 approach. You must:
 
-#. Install git locally. For Debian/Ubuntu, execute:
+#. Install git locally. In Debian or Ubuntu, run the following command:
```

Comment:
I like For better.  ymmv.

Reply:
I'll make a note to myself to take this matter up in more detail in the future, but I find that "for" is a more evocative and ambiguous preposition than "in" is, and it was for exactly that reason that I changed these here fors to these here ises. However, your preference is duly noted and listed in my handwritten list of matters touching language and technical writing that brilliantly display what H. L. A. Hart called "the open texture of language".

As long as I am running on like this, hi from the airport in Qatar.